helpers/sortAndFilterHotels.js

```diff
@@ -0,0 +1,51 @@
+/* jslint node: true */
+'use strict';
+
+var _ = require('lodash');
+var sortBy = require('../lib/sortBy');
+
+module.exports = function(dust) {
+
+  /*
+   * @disclaimer - this is a temporary filter helper to separate a recommended hotel(s) from a list of hotels. This is temporary because it's written in dust, which will change soon.
+   *
+   * @description Augmented sorting method that orders a list of things based on a child node, and then filters the result based on an object and a value .
+   * @param {object} parent Object to sort
+   * @param {string} node Child node to sort object by
+   * @param {object} filterArray - array to base the filter on
+   * @param {boolean as string} exclude - if 'true', the filter function will exclude all items in the filterArray, based on the filterValue
```

Comment:
would it make sense to have one variable for both `include` and `exclude` and simply flip the negative? I'm guessing you'd never use both at the same time, the only issue would be if you can pass neither when calling?